front50-core/src/main/groovy/com/netflix/spinnaker/front50/model/application/ApplicationDAO.groovy

```diff
@@ -35,4 +35,46 @@ public interface ApplicationDAO {
 
   Set<Application> search(Map<String, String> attributes)
 
-}
\ No newline at end of file
+  void bulkImport(Collection<Application> applications)
+
+  static class Searcher {
```

Comment:
Nevermind - it doesn't look like there's anything cassandra specific about this.